src/iphone_backup_decrypt/iphone_backup.py

```diff
@@ -271,7 +271,7 @@ def extract_files(self, *, relative_paths_like, output_folder):
             query = """
                 SELECT fileID, relativePath, file
                 FROM Files
-                WHERE relativePath LIKE ?
+                WHERE domain LIKE ?
```

Comment:
```suggestion
                WHERE relativePath LIKE ?
```
I think adding support for `domain` is very sensible, and I probably finally have time to do it. It can probably just go as an optional argument in both existing methods; if no relative paths are specified in this method but a domain is, then export all files for that domain.

But I don't think swapping the `relativePath` to `domain` here is a good idea given that it is still named `relative_paths_like` everywhere in the code and docstrings!
I am happy to merge this if that is reverted, and then I will try and add domain filtering too.